control/autonomous_emergency_braking/src/node.cpp

```diff
@@ -221,9 +223,17 @@ void AEB::onPointCloud(const PointCloud2::ConstSharedPtr input_msg)
     pcl::fromROSMsg(transformed_points, *pointcloud_ptr);
   }
 
+  // apply z-axis filter for removing False Positive points
+  PointCloud::Ptr height_filtered_pointcloud_ptr(new PointCloud);
+  pcl::PassThrough<pcl::PointXYZ> height_filter;
+  height_filter.setInputCloud(pointcloud_ptr);
+  height_filter.setFilterFieldName("z");
+  height_filter.setFilterLimits(detection_range_min_height_, 5.0);
```

Comment:
Ideally, 5.0 should depend on the vehicle's height. How about setting the vehicle's height + ros parameter to this value?
The ros parameter's name would be `detection_range_height_margin` or `detection_range_additional_height`.

Reply:
@takayuki5168 

Thank you for your comment!
I changed this code following your comment in https://github.com/autowarefoundation/autoware.universe/pull/6637/commits/49311e4f68b95854b3803b8dd088463789668802

- Please check if the name of parameter (`detection_range_max_height_margin`) is easy to understand.
- Please check if the default value of parameter (`detection_range_max_height_margin`) 0 is appropriate.